server-test/src/test/scala/testpkg/ServerCompletionsTest.scala

```diff
@@ -33,6 +33,7 @@ class ServerCompletionsTest extends AbstractServerTest {
     })
   }
 
+  /*
```

Comment:
I think you commented this one out because it's flaky. Maybe add `// TODO fix flakiness`?